src/unstract/sdk/adapters/x2text/llm_whisperer/src/llm_whisperer.py

```diff
@@ -310,25 +317,61 @@ def _send_whisper_request(
     def _extract_text_from_response(
         self, output_file_path: Optional[str], response: requests.Response
     ) -> str:
-
-        output = ""
+        output_json = {}
         if response.status_code == 200:
-            output = response.content.decode("utf-8")
+            output_json = response.json()
         elif response.status_code == 202:
-            whisper_hash = response.json().get(WhisperStatus.WHISPER_HASH)
-            output = self._extract_async(whisper_hash=whisper_hash)
+            whisper_hash = response.json().get("WhisperStatus.WHISPER_HASH")
```

Comment:
@Deepak-Kesavan Why are we wrapping `"WhisperStatus.WHISPER_HASH"` inside quotes? Should this be a constant?

Reply:
Thanks for the catch. Fixed it.